pawn/Features/Gameplay/VIP/VeryImportantPlayersCommands.pwn

```diff
@@ -33,7 +33,7 @@ class VeryImportantPlayersCommands {
         }
 
         if (Command->parameterCount(params) != 1) {
-            SendClientMessage(playerId, Color::Information, "Usage: /my weather [day/evening/hot/storm/foggy/drugs]");
+            SendClientMessage(playerId, Color::Information, "Usage: /my weather [day/evening/underwater/sandstorm/drugs/hot/cloudy/storm/foggy]");
```

Comment:
I suggest to keep the order of the first ones the same.

They'll be the most relevant weathers for the players to use, having those as first reads won't cludge it too much.
You could also possibly go for an alphabetical order.

Reply:
Done :)